.github/workflows/link-check.yml

```diff
@@ -16,3 +16,6 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        folder-path: '.,'
```

Comment:
@jarovo the default value is `.` so you can delete this line and it should work.

Reply:
@gaurav-nelson  I removed that line in the commit bellow. The test should fail, but it is still passing. What am I doing wrong? Thanks.

Reply:
@jarovo gaurav-nelson/github-action-markdown-link-check@v1 uses an old version of markdown-link-check that doesn't check for anchors. If you run it with `- uses: gaurav-nelson/github-action-markdown-link-check@v1.0.15` you will see the test fail. I think that should do. I tested locally and it did fail. Can you try with that?